src/NuGet.Core/NuGet.Packaging/Signing/Content/KeyPairFileReader.cs

```diff
@@ -103,4 +103,4 @@ public void Dispose()
             _reader.Dispose();
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
empty line at the end

Reply:
[Coding guidelines](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style), rule 18:
> Do add a new line at the end of all files.